playbook/workshops/dodge/add_player_sprite.md

```diff
@@ -0,0 +1,135 @@
+
+![](img/checkmark.png) Open [the above JS Bin](http://jsbin.com/qiyuno/14/edit?js,output) and try messing around with the starting position of the sprite.
+
+Here is an example of how I play around with the values of `createSprite` to try to understand what it does:
+
+> ![](img/2_messing_around_with_starting_position.gif)
+
+## Understanding `createSprite`
+
+There are 3 parts to creating a sprite:
+
+1. creating a variable to store the sprite in
+2. creating the actual sprite and storing it in the above variable
+3. drawing the sprite
+
+1. First we need to create the variable `player` to store the sprite in:
+
+> ![](img/2_variable.png)
+
+2. Then we actually create the sprite and store it in the variable `player`:
+
+> ![](img/2_create_sprite.png)
+
+```
+player = createSprite(150, 450);
+  ^                    ^    ^
+  |                    |    └ set the *y* coordinate of the sprite
+  |                    |
+  |                    └ set the *x* coordinate of the sprite
+  └ Store the newly
+    created sprite
+    with the variable `player`
+```
+
+> ![](img/t2_sprite_position.png)
+
+3. Then we have to tell the program to actually draw the sprites in our program by
+calling `drawSprites()`:
+
+> ![](img/2_draw_sprites.png)
+
+If we don't include `drawSprites()`, even if the sprites have been created, you
+cannot see them.
+
+## Adding the Sprite to Your Code
+
+![](img/checkmark.png) Now add the 3 lines of code needed to add the sprite to your "_**working bin**_".
+
+> ![](img/t2_add_code.gif)
+
+## Recap
+
+We learned how to:
+
+- create a variable
+- create a new sprite with `createSprite`
+- the new sprite's starting position
+- store a sprite into a variable
+- draw the sprite
+
+## Next Up
+
+| **[![](img/sq_3_linear_player_movement.gif)  <br> 3. Linear Player Movement] (linear_player_movement.md)**    |
+|:--------------------------------------------------------------------------------------------------------------|
+
+--------------------------------------------------------------------------------
+
+<!--
+
+Commented this out because it's not good enough yet.
+
+I want to leave it in the comments because it has some work done on it
+
+## Appendix
+
+### Documentation ![](img/documentation.png)
+
+#### [p5.js Play Documentation](http://p5play.molleindustria.org/docs/index.html)
+
+- [`createSprite(150, 450)`](http://p5play.molleindustria.org/docs/classes/p5.play.html#method-createSprite)
+
+#### Understanding the Computer Science Fundamentals ![](img/computer_science.png)
+
+```
+This section goes over some of the fundamental programming concepts, you can
+feel free to skip this section if you like.
+```
+
+In order for a program to remember something, we must use a variable to
+remember it. And so we create the `player` variable to remember the player sprite.
+
+> ![](img/2_variable.png)
+
+`createSprite is a function` functions have 3 components
+
+- inputs
+- actions
+- output
+
+... -->
```

Comment:
Perhaps it'd make more sense to keep this on a separate branch? Git is great because it lets you do stuff like this :-)